common/src/main/java/com/dtp/common/constant/DynamicTpConst.java

```diff
@@ -61,6 +61,11 @@ private DynamicTpConst() { }
 
     public static final String TASK_WRAPPERS = "taskWrappers";
 
+    /**
+     * alarm
+     */
```

Comment:
The NOTICE_EXECUTOR of NoticeManager can also have a hook added to it.

Reply:
> The NOTICE_EXECUTOR of NoticeManager can also have a hook added to it.

I also optimized NacosRefresher.
PTAL